<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserFeedback::Analyzer::SchemaEditor</class>
 <widget class="QWidget" name="UserFeedback::Analyzer::SchemaEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="UserFeedback::Analyzer::SchemaEditWidget" name="schema">
      <attribute name="title">
       <string>Schema</string>
      </attribute>
     </widget>
     <widget class="UserFeedback::Analyzer::AggregationEditWidget" name="aggregation">
      <attribute name="title">
       <string>Aggregation</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>UserFeedback::Analyzer::SchemaEditWidget</class>
   <extends>QWidget</extends>
   <header>schemaeditor/schemaeditwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>UserFeedback::Analyzer::AggregationEditWidget</class>
   <extends>QWidget</extends>
   <header>schemaeditor/aggregationeditwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
